README.md

```diff
@@ -49,6 +49,36 @@ export default class Button extends React.Component {
 }
 ```
 
+##### tagged `@decorator` syntax
+
+Or directly as a tagged decorator without the need of importing CSJS explicitly.
```

Comment:
I think we should promote this style first, so the "Or" case is importing `csjs` explicitly